torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -151,8 +150,6 @@ def optimized_mod(*args):
                 f"torchxla reuse compiled graph run_cached_graph takes {time.time() - start_ts} seconds"
             )
 
-        prepare_output_ts = time.time()
-
         copy_args_ts = time.time()
```

Comment:
better to rename copy_args_ts to ... idk... inplace_update_ts ?

Reply:
yep, will do

Reply:
Or args_inplace_update_ts to emphasis that we are doing this for arguments :) ?